server/game/cards/03_TWI/events/InPursuit.ts

```diff
@@ -0,0 +1,34 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { RelativePlayer, WildcardCardType } from '../../../core/Constants';
+
+export default class InPursuit extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '6401761275',
+            internalName: 'in-pursuit'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Exhaust a friendly unit. If you do, exhaust an enemy unit.',
```

Comment:
nit: it doesn't yet but the plan is to make things so that the "if you do" part of the title is automatically generated from the title of the `ifYouDo` ability. So it doesn't need to be included here, this can just be `Exhaust a friendly unit`

Reply:
changed